pybikes/gbfs.py

```diff
@@ -127,3 +127,7 @@ def __init__(self, info):
             'returning': info['is_returning'],
             'last_updated': info['last_reported']
         }
+
+        if 'num_ebikes_available' in info:
+            self.extra['has_ebikes'] = int(info['num_ebikes_available']) > 0
```

Comment:
I think this one is always true. Not sure where else has_ebikes is being used right now, but i think it says if the station supports ebikes, no matter the quantity available

Reply:
Thank you for the feedback!  I think you're right. I'm looking at your commit from June https://github.com/eskerda/pybikes/commit/210cb89ab3682517bb67dbf78bc02477835dd4bb and you're setting `has_ebikes` to true if there is an ebike related key in the data from the services. I'll get an update for that shortly.